cli/cmd/resource_apply_cmd.go

```diff
@@ -51,11 +54,26 @@ func init() {
 	}
 
 	applyCmd.Flags().StringVarP(&applyParams.DefinitionFile, "file", "f", "", "path to the definition file")
+	applyCmd.Flags().StringVarP(&applyParams.GitRepo, "gitrepo", "", "", "Git repository name")
+	applyCmd.Flags().StringVarP(&applyParams.GitUsername, "gitusername", "", "", "Git username")
+	applyCmd.Flags().StringVarP(&applyParams.GitToken, "gittoken", "", "", "Git token")
+	applyCmd.Flags().StringVarP(&applyParams.RepoName, "reponame", "", "", "Repository name")
+	applyCmd.Flags().StringVarP(&applyParams.Branch, "branch", "", "", "Branch name")
+	applyCmd.Flags().StringVarP(&applyParams.GitFile, "gitfile", "", "", "Git file name")
+
 	rootCmd.AddCommand(applyCmd)
+	logger.Info("Exiting resource_apply_init()")
```

Comment:
convert it into debug
```suggestion
	logger.Debug("Exiting resource_apply_init()")
```